### PR TITLE
make artpiece grid columns five across

### PR DESCRIPTION
**Pull request by denisdimitrov**

### CODE REVIEW

File: src/components/templates/ArtPiece/ArtPiece.scss [hunk 1]
```diff
@@ -17,7 +17,7 @@
   .video-chat-wrapper {
     display: grid;
     margin: 1rem 0;
-    grid-template-columns: repeat(6, 1fr);
+    grid-template-columns: repeat(9, 1fr);
```
Comment (0xdevalias) [inline src/components/templates/ArtPiece/ArtPiece.scss:20]: Just so I can understand it better, can you explain what this is doing/how it works please?

**Review by 0xdevalias — COMMENTED** [review]
The title of this PR says make the videos 3 across, but I believe that was the existing functionality.

The trello card I can see talks about making them 4 across. Is that what this PR is actually doing?

**Comment by denisdimitrov** [comment]
[Art piece video arrangement only three across (should go to four)](https://trello.com/c/glKdnkQX/452-art-piece-video-arrangement-only-three-across-should-go-to-four)

File: src/components/templates/ArtPiece/ArtPiece.scss (hunk 1 above)
Comment (denisdimitrov) [reply src/components/templates/ArtPiece/ArtPiece.scss]: I just spoke with Ed, and this should be 5 across. 1 video takes 3 columns. Not sure why, but we are in a hurry for the event and I don't have the time to look into it.

Comment (0xdevalias) [reply src/components/templates/ArtPiece/ArtPiece.scss]: No worries, don't need to clean it up now. Was just curious/for my own knowledge.

**Review by 0xdevalias — APPROVED** [review]
Needs to be 5 across, videos apparently take up 3 columns. That matches the 15 in the changes here. LGTM!

**Pull request merged**